pahfit/model.py

```diff
@@ -187,8 +188,13 @@ def guess(
         # parse spectral data
         self.features.meta["user_unit"]["flux"] = spec.flux.unit
         _, _, _, lam, flux, _ = self._convert_spec_data(spec, z)
-        wmin = min(lam)
-        wmax = max(lam)
+        lam_min = min(lam)
+        lam_max = max(lam)
+
+        # Some useful quantities for guessing
+        median_flux = np.median(flux)
+        Flambda = flux * units.intensity * (lam * units.wavelength) ** -2 * constants.c
```

Comment:
Would it be smarter to do "local power" integrals near resolved features?

Reply:
Perhaps if we're careful about it. But when I tried something like that for the dust features, I found that trying to be too smart with the initial guess often results in getting stuck in a local minimum. For the lines I did put in an option for an approximate local power integral though.

Reply:
Sounds good.